src/Metadata/class-date-builder.php

```diff
@@ -47,10 +49,11 @@ private function build_headline(): void {
 			$this->metadata['headline'] = sprintf( __( 'Monthly Archive - %s', 'wp-parsely' ), get_the_time( 'F, Y' ) );
 		} elseif ( is_day() ) {
 			/* translators: %s: Archive day, formatted as F jS, Y */
-			$this->metadata['headline'] = sprintf( __( 'Daily Archive - %s', 'wp-parsely' ), get_the_time( 'F jS, Y' ) );
+			$this->metadata['headline'] = sprintf( __( 'Daily Archive - %s', 'wp-parsely' ), get_the_time( $site_date_format ) );
 		} elseif ( is_time() ) {
+			$site_time_format = get_option( 'time_format' );
 			/* translators: %s: Archive time, formatted as F jS g:i:s A */
-			$this->metadata['headline'] = sprintf( __( 'Hourly, Minutely, or Secondly Archive - %s', 'wp-parsely' ), get_the_time( 'F jS g:i:s A' ) );
+			$this->metadata['headline'] = sprintf( __( 'Hourly, Minutely, or Secondly Archive - %s', 'wp-parsely' ), get_the_time( "{$site_date_format}:{$site_time_format}" ) );
```

Comment:
Whilst it seems that WP core no longer has a default title for `is_time()`, I don't think the use of the `:` between the date and time makes sense here. You might get, for instance, `November 14th:12:40:00`. Too many `:` for my liking. Instead of the `:` between the date and time, consider a space instead.

```suggestion
			$this->metadata['headline'] = sprintf( __( 'Hourly, Minutely, or Secondly Archive - %s', 'wp-parsely' ), get_the_time( "{$site_date_format} {$site_time_format}" ) );
```

The corresponding test(s) will need updating as well.

Reply:
I think that @GaryJones was saying is to add a space instead of a `:` (not adding a space before and after the `:`). But I'll let him clarify on that.

Reply:
@mehmoodak, if there is a specific reason you consider `:` to be a better choice, let us know.

Reply:
You are right. I misread this. Making this change

Reply:
Change incorporated.